providers/flagd/src/test/resources/flagConfigurations/valid-long.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "../../../main/resources/flags.json",
```

Comment:
Adding links to local json validation file so we can see these files validation status live.

Reply:
This is a nice addition 👍